### ISSUE TITLE
Header nav focus order differs from visual order

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/call.zetk.in#278]
## Steps to reproduce

1. Log in.
2. Press `⌘ + F5` to start VoiceOver.
3. Focus the `My page` button in the header nav.

## Expected outcome

The `My page` button is visually before the user menu button so it should be before it in the focus order. See [C27: Making the DOM order match the visual order](https://www.w3.org/TR/WCAG20-TECHS/C27.html) for details of the success criteria this addresses and the assistive technology use cases it supports.

## Actual outcome

The user menu buttons is before the `My page` button in the focus order.

![Screen recording demonstrating the scenario described above.](https://user-images.githubusercontent.com/566159/195791242-4cb1b766-747e-4b32-8832-9c20313dde13.gif)

## Severity

The success criterion covering this issue is WCAG level A – the minimum level. Most organizations aim for a conformance level of AA, which includes all level A criteria. In this particular case this issue seems unlikely to seriously hamper anybody's work but as it's level A it's still moderately severe.

## Recommendation

Render the header nav's DOM elements in the same order as they'll appear visually, and remove the CSS that inverts the visual order.
